packages/vite/src/node/plugins/define.ts

```diff
@@ -59,16 +59,18 @@ export function definePlugin(config: ResolvedConfig): Plugin {
       ...processEnv
     }
 
+    const characters = '[\'"`\\/]'
```

Comment:
```suggestion
    const characters = '[\'"`\\/_-]'
```
For example, the following code
```js
import * from "xxxx/xx(_|-)PATH
```